department-of-veterans-affairs/caseflow
# Correspondence Days Waiting Sort - hotfix

Resolves [Your Correspondence - In Progress Tab](https://jira.devops.va.gov/browse/APPEALS-35479)

# Description
Previously sorting by days waiting was both broken and also sorting in the wrong direction. This change fix the task sorting and also causes it to sort in the correct direction.

## Acceptance Criteria

## Testing Plan
1. Navigate to the correspondence table
2. filter by days waiting. 
3. The days waiting should now be greatest to least, and the up arrow colored blue.
4. Click the days waiting sort button again
5. The days waiting should now be least to greatest, and the down arrow colored blue.
